src/ecalc/libraries/libecalc/common/tests/common/test_time_utils.py

```diff
@@ -88,6 +88,46 @@ def test_three_dates(self):
             ]
         )
 
+    def test_three_dates_not_include_before(self):
+        first_date = datetime(2020, 1, 1)
+        second_date = datetime(2022, 1, 1)
+        third_date = datetime(2022, 1, 1)
```

Comment:
did you intend to set second and third date to the same date? I guess this can happen in the code..but imo. this will instead treat symptom and there is another underlying problem...? Might be that we have chosen to postpone this for v2 due to backwards compatibiltiy etc though...

Reply:
No, I did not. Copy paste of the wrong test, I guess